implementation/src/main/java/io/smallrye/mutiny/operators/AbstractMulti.java

```diff
@@ -115,6 +115,11 @@ public void subscribe(Subscriber<? super T> subscriber) {
         return new MultiOnSubscribe<>(this);
```

Comment:
should is call onSubscription() instead? (so we have a single entry point)

Reply:
yep